src/ng/cacheFactory.js

```diff
@@ -223,6 +223,8 @@ function $CacheFactoryProvider() {
             delete lruHash[key];
           }
 
+          if (!(key in data)) return;
```

Comment:
Could I get a bit more guidance as to how I should make use of `createMap` in this case, though? If I'm setting `data = createMap()`, wouldn't that effectively be clearing out everything that was being held inside of `data`?